base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -615,6 +615,53 @@ impl ConsensusConstants {
         vec![consensus_constants_1]
     }
 
+    pub fn stagenet() -> Vec<Self> {
+        // Note these values are all placeholders for final values
+        let difficulty_block_window = 90;
+        let mut algos = HashMap::new();
+        algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
+            max_target_time: 1800,
+            min_difficulty: 40_000.into(),
+            max_difficulty: u64::MAX.into(),
+            target_time: 300,
+        });
+        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+            max_target_time: 800,
+            min_difficulty: 70_000_000.into(),
+            max_difficulty: u64::MAX.into(),
+            target_time: 200,
+        });
+        let (input_version_range, output_version_range, kernel_version_range) = version_zero();
+        vec![ConsensusConstants {
+            effective_from_height: 0,
+            coinbase_lock_height: 1,
```

Comment:
We have been using small numbers to make tests easier, but I take it we want to have stagenet be a larger more realistic number?

Reply:
Yeah, I did find my test mining was going a little fast. Who needs to make the call on what these final numbers should be?

Reply:
Moved the numbers to match current esme constatns + larger lock height.